actions/ostree_deploy_action.go

```diff
@@ -1,3 +1,46 @@
+/*
+OstreeDeploy Action
+
+Put the selected OSTree branch to target target image.
+If any preparation has been done for rootfs, it can be overwritten
+during this step.
+
+Action 'image-partition' must be called prior to OSTree deploy.
+
+Yaml syntax:
+ - action: ostree-deploy
+   repository: repository name
+   remote_repository: URL
+   branch: branch name
+   os: stateroot
+   setup-fstab: bool
+   setup-kernel-cmdline: bool
+   appendkernelcmdline: arguments
+
+Mandatory properties:
+
+- repository -- path to repository with OSTree structure; the same path is
+used by 'ostree' tool with '--repo' argument.
```

Comment:
Is how the ostree tool gets used relevant here?

Reply:
A kind of a hint for people who is already familiar with ostree itself.
Removed.

Reply:
If you say `path to the ostree repository` anyonje know ostree should understand what you mean :)